### ISSUE TITLE
[RFE] add hostname of affected machine to subject of email notification and remove spare "Alert"

### ISSUE THREAD

**Issue by dahorak** [Tendrl/notifier#137]
It might be useful to add also *hostname* of affected machine into *subject* of email notification.

It will help with orientation in the email flow. For example from the following screenshot it is not immediately clear, if there is only one problematic server or more.

![email_notification_subjects](https://user-images.githubusercontent.com/4759779/32212061-8a9cad32-be15-11e7-9a8b-f49ba8466d11.png)

Also the second *Alert* word is unnecessary and I think it might be omitted.

Actual form:
*Subject:* [Tendrl Alert] Memory Alert, WARNING: threshold breached

Expected form:
*Subject:* [Tendrl Alert] Memory, WARNING: threshold breached - gluster1.example.com

**Comment by GowthamShanmugam** [comment]
@dahorak what about cluster related alert, cluster name is big right, is this good to add cluster name in mail subject for cluster-related alerts? 

@shtripat ^

**Comment by dahorak** [comment]
@GowthamShanmugam I agree, that Gluster "name" is not much meaningful and is quite long.
Maybe it might make sense to use only some part of the name - similarly what git do with commit hash - the hash is long, but on most places, it shows/uses only first few (on github 7) characters.

**Comment by GowthamShanmugam** [comment]
having partial cluster name is not going to help much, what I feel is better we keep the only hostname in the subject only for node related alerts. 

Because in UI also filtering is happening only by integration id, cluster also lister in UI only using  integration id, 
Inside alert message also we are having only integration id. 
so maintaining cluster name in a subject not going to help I feel.

one more problem is brick related alerts are common for node and cluster, so here confusion occurs like which name I have to give in subject

@shtripat @nthomas-redhat @r0h4n @nnDarshan

**Comment by shtripat** [comment]
@GowthamShanmugam I feel this generalization issue is because the `subject` line of notification is formed inside notifier module. I would suggest if possible get the same done within handler, as we have more control over what data to be put in the subject line.

@Tendrl/tendrl-core comments ?

**Comment by r0h4n** [comment]
@brainfunked  Can you please comment on this

**Comment by GowthamShanmugam** [comment]
@Tendrl/tendrl-core

**Comment by brainfunked** [comment]
How about a format such as: `[Tendrl] <PRIORITY>: <alert message> [<Component>] (<HOST/CLUSTER>: <id>)`? This would lead to examples such as:

* `[Tendrl] INFO: Threshold breach [Memory] (HOST: <fqdn>)`
* `[Tendrl] WARNING: Threshold breach [Volume Capacity] (CLUSTER: <cluster id>)`

The use of brackets and terms in a specific pattern would also make the subject lines machine parsable, which may prove to be useful in some cases.

**Comment by r0h4n** [comment]
@GowthamShanmugam updates?
